crystalfontz/client.py

```diff
@@ -611,6 +651,8 @@ async def client(
     firmware_rev: Optional[str] = None,
     device: Optional[Device] = None,
     report_handler: Optional[ReportHandler] = None,
+    timeout: float = float("inf"),
```

Comment:
We can make this 0.25, the overhead of Python, not counting serial itself, is trivial, and even status responds will within that limit.